app-editors/vscode/vscode-1.85.0-r2.ebuild

```diff
@@ -104,10 +104,25 @@ src_install() {
 
 	dosym -r "/opt/${PN}/bin/code" "usr/bin/vscode"
 	dosym -r "/opt/${PN}/bin/code" "usr/bin/code"
-	domenu "${FILESDIR}/vscode.desktop"
-	domenu "${FILESDIR}/vscode-url-handler.desktop"
-	domenu "${FILESDIR}/vscode-wayland.desktop"
-	domenu "${FILESDIR}/vscode-url-handler-wayland.desktop"
+
+	EXEC_EXTRA_FLAGS=""
+	if use wayland-on-wayland; then
+		EXEC_EXTRA_FLAGS+="--ozone-platform-hint=auto "
+	fi
+	if use egl; then
+		EXEC_EXTRA_FLAGS+="--use-gl=egl "
+	fi
+
+	sed "s|@exec_extra_flags@|$EXEC_EXTRA_FLAGS|g" \
+		"${FILESDIR}/vscode-url-handler.desktop" \
+		> "${T}/vscode-url-handler.desktop" || die
```

Comment:
```suggestion
	local EXEC_EXTRA_FLAGS=()
	if use wayland-on-wayland; then
		EXEC_EXTRA_FLAGS+=( "--ozone-platform-hint=auto" )
	fi
	if use egl; then
		EXEC_EXTRA_FLAGS+=( "--use-gl=egl" )
	fi

	sed "s|@exec_extra_flags@|${EXEC_EXTRA_FLAGS[*]}|g" \
		"${FILESDIR}/vscode-url-handler.desktop" \
		> "${T}/vscode-url-handler.desktop" || die
```

Reply:
Thanks! I fixed up the commits with this change